### PR TITLE
Remove AD Desktop Discover Flow

### PR DESCRIPTION
**Pull request by ibeckermayer**
Removes the AD Desktop Discover Flow and the documentation that pointed to it, marking the relevant endpoints as deprecated. Changes the `Active Directory Users` discovery tile to point to the manual documentation (https://goteleport.com/docs/desktop-access/active-directory-manual):

image

### CODE REVIEW

**Comment by ibeckermayer** [comment]
> Isn't there a bunch more web code that can be removed?

🙈

4ba90ea6e5e1308b4ca4818f359af78bd9663cef

File: web/packages/teleport/src/services/userEvent/types.ts [hunk 1]
```diff
@@ -85,9 +85,7 @@ export enum DiscoverEvent {
   DatabaseRegister = 'tp.ui.discover.database.register',
   DatabaseConfigureMTLS = 'tp.ui.discover.database.configure.mtls',
   DatabaseConfigureIAMPolicy = 'tp.ui.discover.database.configure.iampolicy',
-  DesktopActiveDirectoryToolsInstall = 'tp.ui.discover.desktop.activeDirectory.tools.install',
   DesktopActiveDirectoryConfigure = 'tp.ui.discover.desktop.activeDirectory.configure',
-  AutoDiscoveredResources = 'tp.ui.discover.autoDiscoveredResources',
```
Comment (kimlisa) [inline web/packages/teleport/src/services/userEvent/types.ts:90]: i'd put this back, it wasn't made specifically for desktops (though it was the only resource emitting it), but it maybe useful later?

**Review by kimlisa — APPROVED** [review]

**Review by avatus — APPROVED** [review]

File: web/packages/teleport/src/services/userEvent/types.ts (hunk 1 above)
Comment (ibeckermayer) [reply web/packages/teleport/src/services/userEvent/types.ts:90]: I think we should just bring it back if/when we actually need it ([YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)). It looks like I missed the `DesktopActiveDirectoryConfigure` above this, too.

**Comment by zmb3** [comment]
Removed the backport label for v15. This is a big enough change that it should go in a major.

**Pull request merged**